walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -37,89 +37,102 @@
             return Arrays.asList();
         }
 
-        // If there are at least one optional attendee, consider them.
+        // If there is at least one optional attendee, consider them.
         if (!attendeesOptional.isEmpty()) {
-            List<TimeRange> available = considerOptionalAttendees(events, attendeesRequest, attendeesOptional, durationRequest);
+            List<TimeRange> available = considerAttendees(events, attendeesRequest, attendeesOptional, durationRequest);
 
             // If there was no available time range considering the optional attendees, try just considering the required attendees (if not empty).
             if (available.isEmpty() && !attendeesRequest.isEmpty()) {
```

Comment:
sorry for the churn, just noticed another improvement

Here if available is not empty you can just return it immediately. That way the stuff in the else clause can be unindented and you can dedupe the considerAttendees line

Reply:
No problem at all, thank you so much for the advice on this!

I just addressed this in a new commit.